assets/images/social/Tiktok.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1"
```

Comment:
Looks like we can minify this to save 50%. I use this:
https://jakearchibald.github.io/svgomg/

Reply:
Good call. I did not bother initially in case we would change that for another variation of the logo.